docs/source/explanation/index.rst

```diff
@@ -0,0 +1,26 @@
+Explanation
+===========
+
+This section consists of an algorithms explanation and describes additional features that are supported by OTX.
+``algorithms`` section includes a description of all supported algorithms:
+
+1. Description of supported datasets formats for each task;
+2. Available templates and models;
+3. Description of supervised training pipelines;
+4. Incremental learning approach;
+5. Semi-supervised and Self-supervised algorithms;
+6. Additional algorithm-specific features such as tiling method for Instance Segmentation and Rotated Object Detection.
```

Comment:
As I suggested in other comment, I think the #0 would be better to general description of the algo task. Description in supervised pipeline could be naturally included as initial description in #0.

Reply:
BTW, do we need semicolons?